doc/TUTORIAL.md

```diff
@@ -37,8 +37,9 @@ install node and npm: https://nodejs.org/en/download/
    `npm ci`
```

Comment:
Let's keep these two changes separate. You've got #3435 opened to cover the tutorial changes, so let's not duplicate them

Reply:
Thanks @calebcartwright , actually I wanted to avoid the changes made in my local copy previously, but I couldn't. Do you know how to delete those commits?

Reply:
Ah I see. Looks like you did the original tutorial work in #3435 off your local master branch, and then you subsequently created this feature/subreddit-subscriber branch off of your master branch correct?

Reply:
Yes, I created the branch from my last commit instead of the last commit of the master of the original repository. 
Do you want me to remove this PR, and create another one properly?

Reply:
Yeah I think that would be best, thanks! For reference I'd suggest always using feature branches in your forked repositories to simplify this process, especially if you're going to work on separate efforts in parallel. 

I'll keep an eye out for a new PR